controllers/ProfileController.js

```diff
@@ -1,6 +1,22 @@
 const pool = require("../database/db");
 const profileQueries = require("../models/ProfileModel");
 
+const editProfile = async (req, res) => {
+  try {
+    const { displayName, birthDate, bio, userId } = req.body;
```

Comment:
should we provide some order to these variables? like `userId` to keep it consistent, and then the rest?